union-ai-admin/aws/script/generate.py

```diff
@@ -484,6 +484,14 @@ def create_provisioner_policy(role_type):
                         )
                     ],
                 ),
+                Statement(
+                    Effect=Allow,
+                    Action=[
+                        Action("ec2", "DisassociateAddress"),
+                    ],
+                    Resource=["*"],
```

Comment:
Based on [the docs](https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonec2.html) this permission supports scoping down to specific `elastic-ip` and `network-interfaces`.  In the spirit of preventing `*` as much as possible, can we change this to:

```
arn:aws:ec2:${AWS::Region}:${AWS::AccountId}:elastic-ip/*
arn:aws:ec2:${AWS::Region}:${AWS::AccountId}:network-interface/*
```